logrotate/src/LogRotateHelper.ipp

```diff
@@ -403,7 +450,7 @@ bool LogRotateHelper::createCompressedFile(std::string file_name, std::string gz
         return false;
     }
 
-    int N;
+    size_t N;
     while ((N = fread(buffer, 1, buffer_size, input)) > 0) {
         gzwrite(output, buffer, N);
     }
```

Comment:
Not a problem. I actually wanted to ask you about that...
Will do it first thing tomorrow morning it is pretty late here he he.
On 12 May 2016 01:45, "Kjell Hedström" <notifications@github.com> wrote:

> In logrotate/src/LogRotateHelper.ipp
> <https://github.com/KjellKod/g3sinks/pull/23#discussion_r62967514>:
>
> > @@ -403,7 +450,7 @@ bool LogRotateHelper::createCompressedFile(std::string file_name, std::string gz
> >          return false;
> >      }
> >
> > -    int N;
> > +    size_t N;
> >      while ((N = fread(buffer, 1, buffer_size, input)) > 0) {
> >          gzwrite(output, buffer, N);
> >      }
>
> @cstamatopoulos <https://github.com/cstamatopoulos> please see the line
> below also. The result of the if statement below has no impact. Can you in
> your pull request
> adjust that?
>
> if (gzclose(output) != Z_OK);
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/KjellKod/g3sinks/pull/23/files/ebe121cecbffdc160450288406929ae824f6e601#r62967514>
>